xbmc/filesystem/NFSDirectory.h

```diff
@@ -30,11 +30,12 @@ namespace XFILE
       bool Exists(const CURL& url) override;
       bool Remove(const CURL& url) override;
     private:
-      std::vector<std::shared_ptr<CFileItem>> GetServerList();
-      std::vector<std::shared_ptr<CFileItem>> GetDirectoryFromExportList(const CURL& inputURL);
+      std::vector<std::shared_ptr<CFileItem>> GetServerList() const;
+      std::vector<std::shared_ptr<CFileItem>> GetDirectoryFromExportList(
+          const CURL& inputURL) const;
       bool ResolveSymlink(const std::string& dirName,
                           struct nfsdirent* dirent,
-                          std::string& resolvedPath);
+                          std::string& resolvedPath) const;
```

Comment:
I think it would be clearer to make this function `static` instead of `const`.

Reply:
Yes, makes sense. Thanks.